services/openvpn/client.go

```diff
@@ -60,7 +60,9 @@ func (c *Client) Start(options connection.ConnectOptions) error {
 	log.Infof("client config: %v", clientConfig)
 
 	c.natPinger.BindConsumerPort(clientConfig.LocalPort)
-	err = c.natPinger.PingProvider(clientConfig.vpnConfig.RemoteIP, clientConfig.vpnConfig.RemotePort)
+	if clientConfig.vpnConfig.LocalPort > 0 {
```

Comment:
So provider sends `LocalPort == 0` when NAT traversal is disabled? Can we make this clear by moving this technical details to config itself with explanatory method name, i.e.
```suggestion
	if clientConfig.vpnConfig.isPingRequested() {
```